cypress-tests/cypress/support/commands.js

```diff
@@ -111,6 +111,20 @@ Cypress.Commands.add("merchantDeleteCall", (globalState) => {
   });
 });
 
+Cypress.Commands.add("ListConnectorsFeatureMatrixCall", (globalState) => {
+  cy.request({
+    method: "GET",
+    url: `${globalState.get("baseUrl")}/feature_matrix`,
+    headers: {
+      Accept: "application/json",
+    },
+  }).then((response) => {
+    logRequestId(response.headers["x-request-id"]);
+
+    expect(response.body).to.have.property("data").and.not.empty;
+  });
+});
+
```

Comment:
```suggestion
Cypress.Commands.add("ListConnectorsFeatureMatrixCall", (globalState) => {
  baseUrl = globalState.get("baseUrl");
  url = `${baseUrl}/feature_matrix`

  cy.request({
    method: "GET",
    url: url,
    headers: {
      Accept: "application/json",
    },
  }).then((response) => {
    logRequestId(response.headers["x-request-id"]);

    expect(response.body).to.have.property("data").and.not.empty;
  });
});

```

do we not use any apiKey here? Also, add some more validations around `data`.

Reply:
This is an open endpoint, hence no use of apiKey